### PR TITLE
[13.0][MIG] stock_picking_auto_print

### PR DESCRIPTION
**Pull request by Khalid-SerpentCS**
Direct Print
=========
Auto print when DO is ready.

### CODE REVIEW

**Comment by dreispt** [comment]
@Khalid-SerpentCS can you elapse fix the commit messages to match the original ones?

File: stock_picking_auto_print/__init__.py [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# Copyright (C) 2019 IBM Corp.
-# Copyright (C) 2019 Open Source Integrators
```
Comment (dreispt) [inline stock_picking_auto_print/__init__.py]: Don't remove the copyrights please

File: stock_picking_auto_print/models/stock_picking.py [hunk 2]
```diff
@@ -2,45 +2,48 @@
...
+                        and value.get("company_id")[0] == company_id
+                    ):
+                        return value["id"]
         elif country_id:
             for value in stock_picking_reports:
-                if value.get('country_id'):
-                    if value.get('country_id')[0] == country_id:
-                        return value['id']
+                if value.get("country_id"):
+                    if value.get("country_id")[0] == country_id:
```
Comment (dreispt) [inline stock_picking_auto_print/models/stock_picking.py:28]: You could have joined the nested if conditions, like you did above.

File: stock_picking_auto_print/models/stock_picking.py [hunk 3]
```diff
@@ -2,45 +2,48 @@
...
+                        return value["id"]
         elif company_id:
             for value in stock_picking_reports:
-                if value.get('company_id'):
-                    if value.get('company_id')[0] == company_id:
-                        return value['id']
+                if value.get("company_id"):
+                    if value.get("company_id")[0] == company_id:
```
Comment (dreispt) [inline stock_picking_auto_print/models/stock_picking.py:33]: same here

**Review by dreispt — CHANGES_REQUESTED** [review]
Some fixes needed

**Comment by bodedra** [comment]
Ping @Khalid-SerpentCS

**Comment by bodedra** [comment]
Ping @Chandresh-SerpentCS

**Comment by dreispt** [comment]
Superseded by #209

**Pull request closed without merging**